tools/targets/__init__.py

```diff
@@ -675,6 +675,17 @@ def complete(t_self, resources, elf, binf):
         else:
             psoc6_complete(t_self, elf, binf)
 
+    @staticmethod
```

Comment:
It would be much better if this platform-specific stuff did not have to be inside the shared __init__.py file, but this seems to be the way this works. I expect this to be remedied at some point in the future.

Reply:
This is how it has been done for years.  As this PR has been open a while and had now become very urgent, please let Cypress know if there is any critical changes required so it can be merged before the 5.14 code freeze in two weeks.

Reply:
Yes, this was just an observation rather than a request for change on this point.